lib/datadog/statsd.rb

```diff
@@ -335,6 +325,16 @@ def close
     PROCESS_TIME_SUPPORTED = (RUBY_VERSION >= '2.1.0')
     EMPTY_OPTIONS = {}.freeze
 
+    if PROCESS_TIME_SUPPORTED
+      def now
+        Process.clock_gettime(Process::CLOCK_MONOTONIC) # uncovered
+      end
+    else
+      def now
+        Time.now.to_f
```

Comment:
If you could remove the unused # uncovered here please. Forgot it during the refactor.